### PR TITLE
#2340 - ratings position list layout

### PR DESCRIPTION
**Pull request by tatiana-scandi**
Issue: https://github.com/scandipwa/scandipwa/issues/2340

Fix ratings position

### CODE REVIEW

File: packages/scandipwa/src/component/ProductCard/ProductCard.style.scss [hunk 1]
```diff
@@ -165,12 +161,15 @@
...
         border-radius: 1rem;
         letter-spacing: normal;
-        left: 50%;
-        transform: translateX(-50%);
+
+            &_layout_grid {
+                left: 50%;
+                transform: translateX(-50%);
+                bottom: 0;
+            }
```
Comment (yeegor) [inline packages/scandipwa/src/component/ProductCard/ProductCard.style.scss:172]: Please fix indentation here - preserve it among items of the same level.

File: packages/scandipwa/src/component/ProductCard/ProductCard.component.js [hunk 2]
```diff
@@ -316,6 +317,7 @@ export class ProductCard extends PureComponent {
             <div
               block="ProductCard"
               elem="Reviews"
+              mods={ { layout } }
```
Comment (yeegor) [inline packages/scandipwa/src/component/ProductCard/ProductCard.component.js:320]: Please make sure that this change is acknowledged.

Comment (yeegor) [reply packages/scandipwa/src/component/ProductCard/ProductCard.component.js:320]: Also, what other kinds of layout can be there?

**Review by yeegor — CHANGES_REQUESTED** [review]

File: packages/scandipwa/src/component/ProductCard/ProductCard.component.js (hunk 2 above)
Comment (tatiana-scandi) [reply packages/scandipwa/src/component/ProductCard/ProductCard.component.js:320]: @yeegor Yes, this change is intentional. `ProductCard_Reviews` element requires different styling depending on whether it is `grid` or `list` layout. And different stylings are applying depending on this. Another option is to replace such syntax with for more clarity:
```
const modsOption = layout === 'list' ? 'list' : 'grid';
<div
              block="ProductCard"
              elem="Reviews"
              mods={ { layout:  modsOption} }>
```

Should I change code to smth like that?

**Review by yeegor — APPROVED** [review]

**Pull request merged**